internal/Network/DNS/Types/Internal.hs

```diff
@@ -1330,28 +1330,28 @@ defaultResponse = DNSMessage {
   , additional = []
   }
 
--- | A 'DNSMessage' template for NOTIFY queries for the given zone,
--- as per RFC-1996.
+-- | A 'DNSMessage' template for NOTIFY Request queries for the
```

Comment:
Cosmetic, but `queries` is pretty much redundant following `Request`, so I'd just write `Requests` and drop the word `queries`.